### PR TITLE
feat(tuc.editable_service_name): make compatible with ui-kit header

### PR DESCRIPTION
**Pull request by FredericEspiau**
MANAGER-4419

### CODE REVIEW

File: packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html [hunk 1]
```diff
@@ -1,57 +1,82 @@
...
-    class="editable-service-name"
     name="editableServiceNameForm"
-    data-ng-submit="$ctrl.saveServiceName()"
+    data-ng-submit=":: $ctrl.saveServiceName()"
 >
-    <h1>
-        <!-- NOT IN EDITION -->
+    <div
+        data-ng-hide="$ctrl.inEdition"
+        data-ng-class="{ 'd-flex': !$ctrl.inEdition }"
```
Comment (antleblanc) [inline packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html]: ⚠️ You are using helpers from Bootstrap 4 but it's not defined as a dependency nor a peer dependency of this module.

- `d-flex`
- `h-100`
- `px-2`
- `align-middle`

File: packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html [hunk 2]
```diff
@@ -1,57 +1,82 @@
...
-                title="{{$ctrl.getTitle()}}"
             >
-                <i class="oui-icon oui-icon-pen_line" aria-hidden="true"></i>
+                <span
+                    class="oui-icon oui-icon-pen_line oui-icon_small"
+                    aria-hidden="true"
+                ></span>
+                <span
+                    class="sr-only"
+                    data-translate="{{:: 'tuc_editableServiceName_change' | translate }}"
```
Comment (antleblanc) [inline packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html]: Using the `translate` directive is enough.

Please refer to my previous comment regarding snakeCase naming.

```suggestion
                    data-translate="tuc_editableServiceName_change"
```

File: packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html [hunk 3]
```diff
@@ -1,57 +1,82 @@
...
+                class="btn btn-editable px-2 h-100"
+                data-ng-disabled="!editableServiceNameForm.editableServiceNameInput.$valid"
+            >
+                <span
+                    class="oui-icon oui-icon-success oui-icon_small"
+                    aria-hidden="true"
+                ></span>
+                <span
+                    class="sr-only"
+                    data-translate="{{:: 'tuc_editableServiceName_confirm' | translate }}"
```
Comment (antleblanc) [inline packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html]:
```suggestion
                    data-translate="tuc_editableServiceName_confirm"
```

File: packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html [hunk 4]
```diff
@@ -1,57 +1,82 @@
...
+                class="btn btn-editable px-2 h-100"
+                data-ng-click=":: $ctrl.cancelEdition()"
+            >
+                <span
+                    class="oui-icon oui-icon-error oui-icon_small"
+                    aria-hidden="true"
+                ></span>
+                <span
+                    class="sr-only"
+                    data-translate="{{:: 'tuc_editableServiceName_cancel' | translate }}"
```
Comment (antleblanc) [inline packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html]:
```suggestion
                    data-translate="tuc_editableServiceName_cancel"
```

File: packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html [hunk 5]
```diff
@@ -1,57 +1,82 @@
...
-                    <i class="ovh-font ovh-font-wrong"></i>
-                </button>
-            </span>
-        </span>
-    </h1>
-    <em class="align-middle" data-ng-bind="$ctrl.serviceName"></em>
+            <button
+                type="button"
+                class="btn btn-editable px-2 h-100"
+                data-ng-click=":: $ctrl.cancelEdition()"
```
Comment (radireddy) [inline packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html]: one time binding for ng-click?

Comment (FredericEspiau) [reply packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html]: Yeah, I don't care about the return value so I'd better remove the watcher there

**Comment by FredericEspiau** [comment]
@antleblanc instead of being `one-way binding`, the `tucEditableServiceNameTitle`  binding is now `two-way binding`, it's a breaking change for whomever was expecting the value not to change

File: packages/manager/modules/telecom-universe-components/src/editable-service-name/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+  "tuc_editableServiceName_change": "Changer le nom du service",
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/modules/telecom-universe-components/src/editable-service-name/translations/Messages_fr_FR.json]: We had a tech talk about this in October 2019 where we said it didn't matter 😉

[the remark this answers is not in the document]
Comment (antleblanc) [reply packages/manager/modules/telecom-universe-components/src/editable-service-name/translations/Messages_fr_FR.json]: Sorry but to keep consistency and readability, snake Case is more appropriate I would say.

File: packages/manager/modules/telecom-universe-components/package.json [hunk 7]
```diff
@@ -27,6 +27,7 @@
     "angular-translate": "^2.11.0",
     "angular-ui-bootstrap": "~1.3.3",
     "angular-validation-match": "^1.9.0",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
```
Comment (antleblanc) [inline packages/manager/modules/telecom-universe-components/package.json]: Please make sure that hoisted applications are aligned.

```sh
$ yarn why @ovh-ux/ng-ovh-telecom-universe-components
```

Comment (antleblanc) [reply packages/manager/modules/telecom-universe-components/package.json]: And you can also update the `BREAKING CHANGE` message to precise that this module now require a new peerDependency.

File: packages/manager/modules/telecom-universe-components/src/editable-service-name/translations/Messages_fr_FR.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/modules/telecom-universe-components/src/editable-service-name/translations/Messages_fr_FR.json]: I know you feel this way but we all determined together that we could do whatever we wanted :)

**Pull request closed without merging**